runners/core-java/src/main/java/org/apache/beam/runners/core/metrics/MonitoringInfoConstants.java

```diff
@@ -41,6 +41,8 @@ public static final class Urns {
     public static final String FINISH_BUNDLE_MSECS =
         extractUrn(MonitoringInfoSpecs.Enum.FINISH_BUNDLE_MSECS);
     public static final String TOTAL_MSECS = extractUrn(MonitoringInfoSpecs.Enum.TOTAL_MSECS);
+    public static final String USER_LATEST_INT64 = extractUrn(MonitoringInfoSpecs.Enum.USER_LATEST_INT64);
+    public static final String USER_LATEST_DOUBLE = extractUrn(MonitoringInfoSpecs.Enum.USER_LATEST_DOUBLE);
```

Comment:
Do we need this double type somewhere? Just according to this pr, sees we only use the int64.

Reply:
updated